lib/spaceship/tunes/app_analytics.rb

```diff
@@ -0,0 +1,145 @@
+
+        app_views_interval(start_t, end_t)
+      end
+
+      def app_in_app_purchases
+        start_t, end_t = time_last_7_days
+
+        app_in_app_purchases_interval(start_t, end_t)
+      end
+
+      def app_sales
+        start_t, end_t = time_last_7_days
+
+        app_sales_interval(start_t, end_t)
+      end
+
+      def app_paying_users
+        start_t, end_t = time_last_7_days
+
+        app_paying_users_interval(start_t, end_t)
+      end
+
+      def app_installs
+        start_t, end_t = time_last_7_days
+
+        app_installs_interval(start_t, end_t)
+      end
+
+      def app_sessions
+        start_t, end_t = time_last_7_days
+
+        app_sessions_interval(start_t, end_t)
+      end
+
+      def app_active_devices
+        start_t, end_t = time_last_7_days
+
+        app_active_devices_interval(start_t, end_t)
+      end
+
+      def app_active_last_30_days
+        start_t, end_t = time_last_7_days
+
+        app_active_last_30_days_interval(start_t, end_t)
+      end
+
+      def app_crashes
+        start_t, end_t = time_last_7_days
+
+        app_crashes_interval(start_t, end_t)
+      end
+
+      def app_units_interval(start_t, end_t)
+        client.time_series_analytics([apple_id], ['units'], start_t, end_t, "DAY")
+      end
+
+      def app_views_interval(start_t, end_t)
+        client.time_series_analytics([apple_id], ['pageViewCount'], start_t, end_t, "DAY")
+      end
+
+      def app_in_app_purchases_interval(start_t, end_t)
+        client.time_series_analytics([apple_id], ['iap'], start_t, end_t, "DAY")
+      end
+
+      def app_sales_interval(start_t, end_t)
+        client.time_series_analytics([apple_id], ['sales'], start_t, end_t, "DAY")
+      end
+
+      def app_paying_users_interval(start_t, end_t)
+        client.time_series_analytics([apple_id], ['payingUsers'], start_t, end_t, "DAY")
+      end
+
+      def app_installs_interval(start_t, end_t)
+        client.time_series_analytics([apple_id], ['installs'], start_t, end_t, "DAY")
+      end
+
+      def app_sessions_interval(start_t, end_t)
+        client.time_series_analytics([apple_id], ['sessions'], start_t, end_t, "DAY")
+      end
+
+      def app_active_devices_interval(start_t, end_t)
+        client.time_series_analytics([apple_id], ['activeDevices'], start_t, end_t, "DAY")
+      end
+
+      def app_active_last_30_days_interval(start_t, end_t)
+        client.time_series_analytics([apple_id], ['rollingActiveDevices'], start_t, end_t, "DAY")
+      end
+
+      def app_crashes_interval(start_t, end_t)
+        client.time_series_analytics([apple_id], ['crashes'], start_t, end_t, "DAY")
+      end
+
+      def time_last_7_days
+        time = Time.now
+        past = time - (60 * 60 * 24 * 7)
+        end_t   = time.strftime("%Y-%m-%dT00:00:00Z")
+        start_t = past.strftime("%Y-%m-%dT00:00:00Z")
+
+        return start_t, end_t
+      end
+
+      def time_last_30_days
+        time = Time.now
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.